README.md

```diff
@@ -112,6 +112,13 @@ To run tests, run the following command:
 - Twitter: [@Twitter](https://twitter.com/home)
```

Comment:
![image](https://github.com/DianaBeki/school-library/assets/93039359/5fd35719-3460-4f82-92ac-8c5f1fca7901)

- [ ] It would be nice if you fix your twitter link because right now it is not taking us to your twitter account 😨 Kindly fix it to make yourself easily approachable by the recruiters and engineers and CEOs 👍